File: 0/is_anagram/is_anagram.c

#include <stdio.h>
int     is_anagram(char *a, char *b)
{
    int countA = 0;
    int countB = 0;

    while (*a)
    {
        countA += *a;
        a++;
    }
    while (*b)
    {
        countB += *b;
        b++;
    }
    if (countA == countB)
        return (1);
    return (0);
}

int main()
{
    char *a = "hello";
    char *b = "elloh1";

    printf("%i\n", is_anagram(a, b));
}

File: 0/is_anagram/is_anagram.py
def is_anagram(strA, strB):
    countA = 0
    countB = 0
    for ele in strA:
        countA += ord(ele)
    for ele in strB:
        countB += ord(ele)
    if countA == countB:
        return 1
    else:
        return 0

print(is_anagram('hello', 'ello1h'))